blueos_3_ppc64le_ib/spack.yaml

```diff
@@ -0,0 +1,482 @@
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-gcc-8.3.1
+      - spec: spectrum-mpi@10.3.1.03rtm0%gcc@4.9.3
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-gcc-4.9.3
+      - spec: spectrum-mpi@10.3.1.03rtm0%clang@9.0.0
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-9.0.0
+      - spec: spectrum-mpi@10.3.1.03rtm0%clang@ibm.9.0.0
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-ibm-2019.10.03
+      - spec: spectrum-mpi@10.3.1.03rtm0%clang@ibm.10.0.1
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-ibm-10.0.1
+      - spec: spectrum-mpi@10.3.1.03rtm0%xl@16.1.1.7
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-2020.03.18
+      - spec: spectrum-mpi@10.3.1.03rtm0%xl@16.1.1.10
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-2021.09.22
+      - spec: spectrum-mpi@10.3.1.03rtm0%xl@beta2019.06.20
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-beta-2019.06.20
+      buildable: false
+    python:
+      buildable: false
+      version: [3.8.2]
+      externals:
+      - spec: python@3.8.2
+        prefix: /usr/tce/packages/python/python-3.8.2
+  compilers:
+  - compiler:
+      spec: clang@3.9.1
+      paths:
+        cc: /usr/tcetmp/packages/clang/clang-3.9.1/bin/clang
+        cxx: /usr/tcetmp/packages/clang/clang-3.9.1/bin/clang++
+        fc: /usr/tce/packages/xl/xl-beta-2019.06.20/bin/xlf2003_r
+        f77: /usr/tce/packages/xl/xl-beta-2019.06.20/bin/xlf_r
+      flags: {}
+      operating_system: rhel7
+      target: ppc64le
+      modules: []
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: clang@4.0.0
+      paths:
+        cc: /usr/tcetmp/packages/clang/clang-4.0.0/bin/clang
+        cxx: /usr/tcetmp/packages/clang/clang-4.0.0/bin/clang++
+        fc: /usr/tce/packages/xl/xl-beta-2019.06.20/bin/xlf2003_r
+        f77: /usr/tce/packages/xl/xl-beta-2019.06.20/bin/xlf_r
+      flags: {}
+      operating_system: rhel7
+      target: ppc64le
+      modules: []
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: clang@9.0.0
+      paths:
+        cc: /usr/tce/packages/clang/clang-9.0.0/bin/clang
+        cxx: /usr/tce/packages/clang/clang-9.0.0/bin/clang++
+        fc: /usr/tce/packages/xl/xl-2020.03.18/bin/xlf2003_r
+        f77: /usr/tce/packages/xl/xl-2020.03.18/bin/xlf_r
+      flags: {}
+      operating_system: rhel7
+      target: ppc64le
+      modules: []
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: clang@10.0.1
+      paths:
+        cc: /usr/tce/packages/clang/clang-10.0.1/bin/clang
+        cxx: /usr/tce/packages/clang/clang-10.0.1/bin/clang++
+        fc: /usr/tce/packages/xl/xl-2020.09.17/bin/xlf2003_r
+        f77: /usr/tce/packages/xl/xl-2020.09.17/bin/xlf_r
+      flags: {}
```

Comment:
Is there a reason none of these clang specs point at the non-default gnu standard library via `--gcc-toolchain`? The system default on blueosand toss3 is 4.9.3

Reply:
The present PR is based upon the current state of radiuss-spack-configs: we used to define the gcc-toolchain in the spec.

I am changing that in https://github.com/LLNL/radiuss-spack-configs/pull/49, which is a major update for us. We still have some issues, and the current maintenance shower is not helping with the testing.

Reply:
Note that I created this to test that the new env would be working for us.
In practice, I will likely merge #49 first, and then redo the above.